test/integration/CMakeLists.txt

```diff
@@ -5,7 +5,6 @@ set(dri_tests
   camera.cc
   depth_camera.cc
   distortion_camera.cc
-  gpu_lidar_sensor.cc
```

Comment:
Hmm are the modifications causing the tests to fail?

Reply:
I vaguely remember Kristjan saying something along that line, but actually I have not tested myself...let me try.

Reply:
Yeah there is a bunch of errors, many of them repeated. 

```
/root/gazebo/gz-sensors/test/integration/gpu_lidar_sensor.cc:710: Failure
Expected equality of these values:
  sensor->Range(j * sensor->RayCount() + i)
    Which is: 0
  gz::math::INF_D
    Which is: inf
[  FAILED  ] GpuLidarSensor/GpuLidarSensorTest.VerticalLidar/ogre2, where GetParam() = "ogre2" (491 ms)
[ RUN      ] GpuLidarSensor/GpuLidarSensorTest.ManualUpdate/ogre2
Warning [Utils.cc:130] [/sdf/model[@name="m1"]/link[@name="link1"]/sensor[@name="TestGpuLidar1"]/updateRate:<data-string>:L1]: XML Element[updateRate], child of element[sensor], not defined in SDF. Copying[updateRate] as children of [sensor].
Warning [Utils.cc:130] [/sdf/model[@name="m1"]/link[@name="link1"]/sensor[@name="TestGpuLidar1"]/alwaysOn:<data-string>:L1]: XML Element[alwaysOn], child of element[sensor], not defined in SDF. Copying[alwaysOn] as children of [sensor].
Warning [Utils.cc:130] [/sdf/model[@name="m1"]/link[@name="link1"]/sensor[@name="TestGpuLidar2"]/updateRate:<data-string>:L1]: XML Element[updateRate], child of element[sensor], not defined in SDF. Copying[updateRate] as children of [sensor].
Warning [Utils.cc:130] [/sdf/model[@name="m1"]/link[@name="link1"]/sensor[@name="TestGpuLidar2"]/alwaysOn:<data-string>:L1]: XML Element[alwaysOn], child of element[sensor], not defined in SDF. Copying[alwaysOn] as children of [sensor].
/root/gazebo/gz-sensors/test/integration/gpu_lidar_sensor.cc:831: Failure
The difference between sensor1->Range(mid) and expectedRangeAtMidPointBox1 is inf, which exceeds 1e-4, where
sensor1->Range(mid) evaluates to inf,
expectedRangeAtMidPointBox1 evaluates to 0.5, and
1e-4 evaluates to 0.0001.
/root/gazebo/gz-sensors/test/integration/gpu_lidar_sensor.cc:834: Failure
Expected equality of these values:
  sensor1->Range(last)
    Which is: 0
  gz::math::INF_D
    Which is: inf
/root/gazebo/gz-sensors/test/integration/gpu_lidar_sensor.cc:838: Failure
The difference between sensor2->Range(0) and expectedRangeAtMidPointBox1 is inf, which exceeds 1e-4, where
sensor2->Range(0) evaluates to inf,
expectedRangeAtMidPointBox1 evaluates to 0.5, and
1e-4 evaluates to 0.0001.
/root/gazebo/gz-sensors/test/integration/gpu_lidar_sensor.cc:842: Failure
Expected equality of these values:
  sensor2->Range(last)
    Which is: 0
  gz::math::INF_D
    Which is: inf
[  FAILED  ] GpuLidarSensor/GpuLidarSensorTest.ManualUpdate/ogre2, where GetParam() = "ogre2" (541 ms)
[ RUN      ] GpuLidarSensor/GpuLidarSensorTest.Topic/ogre2
[Msg] Loading plugin [gz-rendering-ogre2]
Warning [Utils.cc:130] [/sdf/model[@name="m1"]/link[@name="link1"]/sensor[@name="TestGpuLidar"]/updateRate:<data-string>:L1]: XML Element[updateRate], child of element[sensor], not defined in SDF. Copying[updateRate] as children of [sensor].
Warning [Utils.cc:130] [/sdf/model[@name="m1"]/link[@name="link1"]/sensor[@name="TestGpuLidar"]/alwaysOn:<data-string>:L1]: XML Element[alwaysOn], child of element[sensor], not defined in SDF. Copying[alwaysOn] as children of [sensor].
[Dbg] [Lidar.cc:133] Laser scans for [TestGpuLidar] advertised on [/lidar]
[Dbg] [GpuLidarSensor.cc:187] Lidar points for [TestGpuLidar] advertised on [/lidar/points]
Warning [Utils.cc:130] [/sdf/model[@name="m1"]/link[@name="link1"]/sensor[@name="TestGpuLidar"]/updateRate:<data-string>:L1]: XML Element[updateRate], child of element[sensor], not defined in SDF. Copying[updateRate] as children of [sensor].
Warning [Utils.cc:130] [/sdf/model[@name="m1"]/link[@name="link1"]/sensor[@name="TestGpuLidar"]/alwaysOn:<data-string>:L1]: XML Element[alwaysOn], child of element[sensor], not defined in SDF. Copying[alwaysOn] as children of [sensor].
[Dbg] [Lidar.cc:133] Laser scans for [TestGpuLidar] advertised on [/topic_with_spaces/characters]
[Dbg] [GpuLidarSensor.cc:187] Lidar points for [TestGpuLidar] advertised on [/topic_with_spaces/characters/points]
Warning [Utils.cc:130] [/sdf/model[@name="m1"]/link[@name="link1"]/sensor[@name="TestGpuLidar"]/updateRate:<data-string>:L1]: XML Element[updateRate], child of element[sensor], not defined in SDF. Copying[updateRate] as children of [sensor].
Warning [Utils.cc:130] [/sdf/model[@name="m1"]/link[@name="link1"]/sensor[@name="TestGpuLidar"]/alwaysOn:<data-string>:L1]: XML Element[alwaysOn], child of element[sensor], not defined in SDF. Copying[alwaysOn] as children of [sensor].
[Err] [Sensor.cc:298] Failed to set sensor topic [@@@]
[Err] [SensorFactory.hh:150] Failed to load sensor [TestGpuLidar] of type[gpu_lidar]
[Err] [Manager.hh:80] Failed to create sensor.
[       OK ] GpuLidarSensor/GpuLidarSensorTest.Topic/ogre2 (430 ms)
```

Reply:
Seems like a failed test `GpuLidarSensor/GpuLidarSensorTest.ManualUpdate/ogre2` having to do with a benign error of setting the topic `[Err] [Sensor.cc:298] Failed to set sensor topic [@@@]`

Reply:
could you please comment out the line and add a comment next to it instead of deleting?

Reply:
seems like there is a bunch of stuff failing. I guess this is passed down from before. I created a task to fix this because I think it's important to pass all tests... https://app.clickup.com/t/863gvzctk

Reply:
Commented out and give comment, pointing to the task. 👍